### PR TITLE
Fixes #193 - Make CONTRIBUTING.md more contribution friendly

### LINKED ISSUE #193: Improve CONTRIBUTING.md
I think we can make the current CONTRIBUTING.md file a bit more "user friendly" by explaining expectations on issues and PRs.

### PR DESCRIPTION
**Pull request by miketaylr**
PTAL @yoavweiss, thanks!

### CODE REVIEW

**Comment by miketaylr** [comment]
(also @amtunlimited -- more eyes on this is a good thing!)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,13 +1,60 @@
...
+[list of other open issues](https://github.com/WICG/ua-client-hints/issues) in
+case one already exists. Comments to existing issues are a great way to
+contribute.
+* Choose a descriptive title when creating a new issue
+
+## Pull Requests
+
+Pull requests to this specification are encouraged and welcome.
+
+Before doing the work to update the spec, please start with a new issue so we
```
Comment (amtunlimited) [inline CONTRIBUTING.md:24]: Should they start a new issue if there's already an existing one? Or assign one to themselves/request to be assigned?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,13 +1,60 @@
...
+can ensure that your changes align with the goals of this specification (or so
+you can explain why the goals of this specification should change, if that's the
+case).
+
+Please note the next section about CLA requirements before any PR can be
```
Comment (amtunlimited) [inline CONTRIBUTING.md:29]: The CLA section is no longer next ;)

**Review by amtunlimited — APPROVED** [review]
A couple of nuts, but LGTM otherwise

File: CONTRIBUTING.md (hunk 2 above)
Comment (miketaylr) [reply CONTRIBUTING.md:29]: Heh, good catch. I promise it was when I wrote that!

File: CONTRIBUTING.md (hunk 1 above)
Comment (miketaylr) [reply CONTRIBUTING.md:24]: Good comments, I can tweak some language to address.

**Review by yoavweiss — APPROVED** [review]
LGTM!!

**Comment by miketaylr** [comment]
Thanks all!

**Pull request merged**